src/ui/FileViewer/DeleteButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FileViewerComponentProps, ViewerTypes } from './types';
+import Icon, { IconColors, IconTypes } from '../Icon';
+
+export function DeleteButton(props: FileViewerComponentProps): React.ReactElement {
+  if (props.viewerType !== ViewerTypes.MULTI) {
+    const { onDelete, isByMe, disableDelete } = props;
+    return (isByMe)
+      ? (
+        <div className="sendbird-fileviewer__header__right__actions__delete">
```

Comment:
How about getting the className from props?
It will be easier when we will debug

And we usually put two class names
which one is getting from props, and the other one is that the UI component has
ex.
```
const { className } = props

<div className={`sendbird-file-viewer-delete-button ${className}`} >
```